src/html/tree.ml

```diff
@@ -87,7 +87,8 @@ let page_creator ~config ~url ~uses_katex name header toc content =
               var content = el.textContent;
               var new_el = document.createElement("span");
               new_el.setAttribute("class", "odoc-katex-math-rendered");
-              katex.render(content, new_el, { throwOnError: false });
+              var display = el.classList.contains("display");
```

Comment:
I should have mentioned - we'll need to add this `display` class in the html renderer.